modules/rosa-using-spot-instance.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/rosa-managing-worker-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-using-spot-instance_{context}"]
+= Adding a machine pool with spot instances to a cluster
+
+After a machine pool is created, you can configure additional machine pools with spot instances.
+
+.Procedure
+
+ifdef::openshift-rosa[]
+. To configure a machine pool with a spot instance, enter the following command:
++
+[source,terminal]
+----
+$ rosa create machinepool --help | grep spot
+  # Add a machine pool with spot instances to a cluster
+  rosa create machinepool -c mycluster --name=mp-1 --replicas=2 --instance-type=r5.2xlarge --use-spot-instances \
+    --spot-max-price=0.5
+      :
+      --spot-max-price string   Max price for spot instance. If empty use the on-demand price. (default "on-demand")
+      --use-spot-instances      Use spot instances for the machine pool.
+----
+endif::[]
```

Comment:
The command is not this whole thing.  This is just showing that the spot flags exist in the `create machine pool` command.  

We should be saying that you add the additional flags to use spot instance.  We can say something like:
In order to use spot instances with your machinepool you must pass in the `--use-spot-instances` flag and if you want to define a max price you can add the `--spot-max-price` flag.  An example is:

rosa create machinepool --cluster=<cluster-name> \
--name=<machinepool-name> \
--replicas=<number-nodes> \
**--use-spot-instances \
--spot-max-price=0.5**